frontend/public/components/app.jsx

```diff
@@ -129,9 +129,15 @@ class App extends React.PureComponent {
         <GlobalNotifications />
         <Switch>
           <Route path={['/all-namespaces', '/ns/:ns',]} component={RedirectComponent} />
+
+          <LazyRoute path="/catalog/all-namespaces" exact loader={() => import('./catalog' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage)} />
+          <LazyRoute path="/catalog/ns/:ns" exact loader={() => import('./catalog' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage)} />
+          <Route path="/catalog" exact component={NamespaceRedirect} />
```

Comment:
One minor thing, the `Catalog` nav-link should be below/after the `Status` per F2F discussions in Raleigh. Thanks!
<img width="322" alt="screen shot 2018-09-18 at 11 03 38 am" src="https://user-images.githubusercontent.com/5903705/45707380-8a21d780-bb33-11e8-855c-31700db602e8.png">

Reply:
Moved, thanks